internal/impl/flakes.go

```diff
@@ -14,22 +14,20 @@ func (d *Devbox) flakeInputs() []*plansdk.FlakeInput {
 	inputs := map[string]*plansdk.FlakeInput{}
 	for _, p := range d.packages() {
 		pkg := nix.InputFromString(p, d.lockfile)
-		if pkg.IsFlake() {
-			AttributePath, err := pkg.PackageAttributePath()
-			if err != nil {
-				panic(err)
-			}
-			if input, ok := inputs[pkg.URLForInput()]; !ok {
-				inputs[pkg.URLForInput()] = &plansdk.FlakeInput{
-					Name:     pkg.Name(),
-					URL:      pkg.URLForInput(),
-					Packages: []string{AttributePath},
-				}
-			} else {
-				input.Packages = lo.Uniq(
-					append(inputs[pkg.URLForInput()].Packages, AttributePath),
-				)
+		AttributePath, err := pkg.PackageAttributePath()
```

Comment:
To clarify: everything will be a flake input now, right? 

Perhaps we could add a function comment here, or maybe in the `plansdk.FlakeInput` file explaining that we can treat user-specified packages and explicit flake inputs uniformly as flakeInputs. Or something along those lines.

For some later cleanup: should this function be renamed `Inputs()`, and and `plansdk.FlakeInput` be renamed `plansdk.Input`, by dropping the explicit "flake" keyword?
By having "flake" keyword be in the name, the question arises: what are the other kinds of inputs?

Reply:
On second thought, I think `flakeInputs()` makes sense (it's not inputs that are flakes, but inputs to a flake).

I think `plansdk. FlakeInput ` could be renamed to `flake.Input` maybe.